CorvallisBus.Core/WebClients/ServiceAlertsClient.cs

```diff
@@ -31,7 +31,7 @@ private static ServiceAlert ParseRow(HtmlNode row)
             var anchor = row.Descendants("a").First();
             var relativeLink = anchor.Attributes["href"].Value;
             var link = new Uri(FEED_URL, relativeLink).ToString();
-            var title = anchor.InnerText;
+            var title = HtmlEntity.DeEntitize(anchor.InnerText);
```

Comment:
This is a tiny fix for a bug where `'` would show up as `&#039;`, for example, in service alert titles.